### PR TITLE
feat: modify workspace role binding

### PR DESCRIPTION
**Pull request by skdud4659**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci`, minor refactoring, etc.)
- [ ] Need discussion
- [ ] Not that difficult
- [ ] Approved feature branch merge to master

### Description
SSIA

### Things to Talk About
Plz check by commits

### CODE REVIEW

File: apps/web/src/services/administration/components/UserManagementFormModal.vue [hunk 1]
```diff
@@ -90,40 +90,58 @@ const setForm = () => {
...
     }
+    return {
+        ...baseRoleParams,
+        workspace_id: item?.name || '',
+    };
+};
 
+/* API */
+const handleConfirm = async () => {
+    console.log('confirm!');
```
Comment (yuda110) [inline apps/web/src/services/administration/components/UserManagementFormModal.vue:127]: nitpick: here!

**Review by yuda110 — APPROVED** [review]
LGTM!! There's many `console.log`😅

File: apps/web/src/services/administration/components/UserManagementFormModal.vue [hunk 2]
```diff
@@ -90,40 +90,58 @@ const setForm = () => {
...
+    const baseRoleParams = {
+        role_id: formState.role.name || '',
     };
-    if (state.data.auth_type === 'LOCAL') {
-        userInfoParams.reset_password = formState.passwordType === PASSWORD_TYPE.RESET;
-    }
-
-    if (state.isChangedToggle) {
-        await fetchPostDisableMfa();
+    console.log(isEmpty(formState.role));
```
Comment (sulmoJ) [inline apps/web/src/services/administration/components/UserManagementFormModal.vue]: nitpick: Hi!

File: apps/web/src/services/administration/components/UserManagementFormModal.vue [hunk 3]
```diff
@@ -90,40 +90,58 @@ const setForm = () => {
...
     try {
-        await fetchRoleBinding();
+        if (state.isChangedToggle) {
+            await fetchPostDisableMfa();
+        }
+
+        if (state.isSetAdminRole) {
+            await fetchRoleBinding();
+        } else {
+            console.log('all');
```
Comment (sulmoJ) [inline apps/web/src/services/administration/components/UserManagementFormModal.vue]: nitpick: Hi Hi

**Review by sulmoJ — APPROVED** [review]
LGTM!

File: apps/web/src/services/administration/components/UserManagementFormModal.vue (hunk 3 above)
Comment (piggggggggy) [reply apps/web/src/services/administration/components/UserManagementFormModal.vue]: kkkk

**Pull request merged**